client/components/happiness-support/README.md

```diff
@@ -0,0 +1,27 @@
+Happiness Support
+===============
+
+This component renders a card for presenting our Happiness Engineer support, including buttons linking to support resources. It can  also render a button to open a Live chat window if the prop `showLiveChatButton` is `true`. 
```

Comment:
I don't understand this description very well. `Happiness Engineer support` sounds like it _is_ live chat, but maybe it's just support documentation. Could this be clarified?